cmd/call-endpoint/main_test.go

```diff
@@ -22,7 +22,7 @@ func TestCallEndpoint(test *testing.T) {
 		// Errors.
 		{
 			CommonCMDTestCase: cmd.CommonCMDTestCase{
-				ExpectedStderrSubstring: `Failed to find the endpoint. | {"endpoint":"ZZZ"}`,
+				ExpectedStderrSubstring: `Failed to find the endpoint. See --list to view all endpoints. | {"endpoint":"ZZZ"}`,
```

Comment:
s/See/Use/

"See" really only makes sense if your are looking up or referencing something.
Here you are invoking the executable again with a new flag.